pom.xml

```diff
@@ -27,7 +27,7 @@
     <groupId>com.eatthepath</groupId>
     <artifactId>pushy-parent</artifactId>
     <packaging>pom</packaging>
-    <version>0.15.2-SNAPSHOT</version>
+    <version>0.15.3-SNAPSHOT</version>
```

Comment:
I appreciate the thought about which release this might go in, but we generally increment this as part of the release process. Let's leave this out of the scope of this pull request.